example/basic_example.py

```diff
@@ -11,8 +11,11 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import platform
+
 from opentelemetry._metrics.measurement import Measurement
 from opentelemetry.sdk._metrics.export import PeriodicExportingMetricReader
+from opentelemetry.sdk.resources import Resource
```

Comment:
Not used right?

Reply:
Yes, thanks for catching that. slightly_smiling_face Removed in [e2f3a29](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/22/commits/e2f3a29cfbef67b8abee561aa685b55d1b7a2d1c)